src/object/cli_ec.c

```diff
@@ -1144,9 +1146,10 @@ obj_ec_recx_reasb(daos_iod_t *iod, d_sg_list_t *sgl,
 					D_ASSERT(reasb_req->orr_size_fetched);
 					iod_size = 1;
 				}
-				ec_data_seg_add(recx, iod_size, sgl, &iov_idx,
-						&iov_off, oca, iovs, iov_nr,
-						sorter, update);
+				ec_data_seg_add(recx, iod_size, sgl,
+						&iov_idx, &iov_off, oca,
+						iovs, iov_nr, sorter,
+						update);
```

Comment:
just confirm why you need to change this, the original indent is incorrect?

Reply:
Original indents were correct.  I'll update the patch.

Reply:
It's the line breaks, not the indents.  I put it back to how it was before I touched this file.